src/Compilers/CSharp/Portable/Syntax/SyntaxExtensions.cs

```diff
@@ -129,6 +129,7 @@ public static SyntaxTriviaList ToSyntaxTriviaList(this IEnumerable<SyntaxTrivia>
 
         internal static XmlNameAttributeElementKind GetElementKind(this XmlNameAttributeSyntax attributeSyntax)
         {
+            RoslynDebug.Assert(attributeSyntax.Parent is object);
```

Comment:
Shouldn't be necessary.

Reply:
Will switch to to `Debug.Assert` (assuming that's what you meant)

---
In reply to: [405844872](https://github.com/dotnet/roslyn/pull/43025#discussion_r405844872) [](ancestors = 405844872)